fl4health/utils/load_data.py

```diff
@@ -94,6 +94,35 @@ def load_mnist_data(
     return train_loader, validation_loader, num_examples
 
 
+def load_mnist_test_data(
+    data_dir: Path,
+    batch_size: int,
+    sampler: Optional[LabelBasedSampler] = None,
+    transform: Optional[Callable] = None,
+) -> Tuple[DataLoader, Dict[str, int]]:
+    """Load MNIST test set only."""
```

Comment:
Maybe we can add true doc strings for this guy since it's net-new? The documentation is also likely easily transferable to the regular `load_mnist_data` function as well if you don't mind?